### PR TITLE
Update pyhmmer to 0.10.6

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`pyhmmer`](https://bioconda.github.io/recipes/pyhmmer/README.html): **0.10.5** &rarr; **0.10.6**

[install with bioconda](http://bioconda.github.io/recipes/pyhmmer/README.html) [Conda](https://anaconda.org/bioconda/pyhmmer/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/pyhmmer`](https://github.com//bioconda/bioconda-recipes/tree/bump/pyhmmer/recipes/pyhmmer) (click to view/edit other files)
Summary | Cython bindings and Python interface to HMMER3.
Home | [https://github.com/althonos/pyhmmer](https://github.com/althonos/pyhmmer)
Releases |[https://pypi.org/pypi/pyhmmer/json](https://pypi.org/pypi/pyhmmer/json)
Recipe Maintainer(s) | @althonos

***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Review by kbseah — APPROVED** [review]
looks good to me

**Comment by kbseah** [comment]
oops don't merge this, just saw #45910 is still work in progress

**Comment by althonos** [comment]
You can merge this one, i still have an issue with test dependencies in #45910 so I'll update there when i make a new release but in the meantime I'd like `v0.10.6` to be available on Bioconda

**Comment by kbseah** [comment]
Thanks for the info! Waiting for a second pair of eyes then

**Pull request merged**
